lib/config/options/index.ts

```diff
@@ -129,6 +129,9 @@ const options: RenovateOptions[] = [
     stage: 'repository',
     type: 'boolean',
     default: false,
+    experimental: true,
+    experimentalDescription:
+      'Config migration PRs are still being improved, in particular to reduce the amount of reordering and whitespace changes.',
```

Comment:
```suggestion
      'We're improving config migration PRs, focusing on reducing the amount of reordering and whitespace changes. We recommend you avoid this feature for now.',
```

How about something like this? We should probably warn our users to stay away while we work on this feature some more. 😄 

We should also be clear on who's doing the work. 😉

Reply:
After this PR is complete, I don't want people to avoid it. In fact I would like for people to test it out. I just want to make sure they don't go enabling it org-wide without thought

Reply:
```suggestion
      'We're improving config migration PRs, focusing on reducing the amount of reordering and whitespace changes.',
```

Let's drop the "stay away, here be dragons" warning then.